lib/subliminal/providers/legendastv.py

```diff
@@ -143,6 +143,9 @@ def get_matches(self, video, hearing_impaired=False):
             if video.imdb_id and self.imdb_id == video.imdb_id:
                 matches.add('imdb_id')
 
+        # archive name
```

Comment:
This was already removed upstream
https://github.com/Diaoul/subliminal/blob/develop/subliminal/providers/legendastv.py